pkg/service/repair/preference.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2023 ScyllaDB
+
+package repair
+
+import "strings"
+
+// TablePreference describes partial predefined order in which tables should be repaired.
+type TablePreference interface {
+	// KSLess compares priorities of two keyspaces according to preference.
+	KSLess(ks1, ks2 string) bool
+	// TLess compares priorities of two tables from the same keyspace according to preference.
+	TLess(ks, t1, t2 string) bool
+}
+
+// internalTablePreference orders internal tables before user tables.
+// It additionally orders internal tables by their importance.
+type internalTablePreference []Unit
+
+func NewInternalTablePreference() TablePreference {
+	return internalTablePreference{
+		{
+			Keyspace: "system_auth",
+			Tables: []string{
+				"role_attributes",
+				"role_members",
+			},
+		},
+		{Keyspace: "system_traces"},
```

Comment:
nit: system_traces can be the last, it's less important than any other table.

Reply:
done